releasenotes/notes/cutqc-package-98dc0e9a3f81ff16.yaml

```diff
@@ -0,0 +1,17 @@
+---
+deprecations:
+  - |
+    The ``circuit_knitting_toolbox.circuit_cutting.wire_cutting`` package
+    is deprecated and should be replaced by the
+    ``circuit_knitting_toolbox.circuit_cutting.cutqc`` package.
```

Comment:
I think we should make clear that -- at least for now -- the _import location_ is deprecated, as the code has moved to a new place.  The package itself is not deprecated; it has just been renamed.

Otherwise, great release note.